.env.template

```diff
@@ -1,3 +1,4 @@
+DEFAULT_NETWORK=
 ALCHEMY_KEY=
```

Comment:
@acuarica how do you switch from one network to the other? Saw on hh that there is a defaultNetwork field on the config for doing this...

Reply:
When `defaultNetwork` is not set, it defaults to `hardhat`.

Usually what I do, instead of configure the default network, I use `hardhat` as default. Then, all `hardhat` scripts defined in `package.json` accept a `network` argument at the end. For instance, you can run a deploy script with

```sh
yarn deploy --network kovan
```

Reply:
However, if it makes sense to have different teams members working on different networks, then I would add this `DEFAULT_NETWORK` config.

Reply:
TBH, I'm not so keen on this workflow. I'm guessing that first we need to solve https://github.com/Ethernautas/renpool/issues/62, and decide how to test again a non-forked test network, _e.g.,_ `kovan`.

Reply:
okok, np, I'll leave the config to you. I'll clode the PR

Reply:
Ok, I'll refactor the config then.